### DISCUSSION TITLE
Networking

### DISCUSSION THREAD

**Discussion by JDPKSP** [picocomputer/community#101] [Q&A]
Hi all,

Since there now exist a pi pico W could it be possible to add networking to the picocomputer? I realize that shared SPI pins might be an obstacle.
That being said, it would be neat to be able to load roms from external repositories (either web or NAS) or perhaps the ability to use SSH. Exposing a networking interface to the 6502 would definitely be neat.

**Comment by rumbledethumps** [comment]
The hardware is ready for this. The software effort needed is massive since there's no free RAM.
